lstchain/reco/reconstructorCC.py

```diff
@@ -204,9 +204,9 @@ def linval(a, b, x):
     return y
 
 
-@njit()
-@cc.export('template_interpolation', 'f8[:,:](b1[:],f8[:,:],f8,f8,f8[:],f8[:],i8)')
-def template_interpolation(gain, times, t0, dt, a_hg, a_lg, size):
+@njit(cache=True)
```

Comment:
Looking at the numba documentation, I never see jit applied to an exported function. What is the effect of applying `njit` to a `cc.export` function?

Reply:
It is needed for call to other functions to work in compiled functions.
I had errors and found this information here https://stackoverflow.com/questions/69783252/numba-ahead-of-time-aot-compiling-with-dependent-functions